.github/workflows/moving-cards.yml

```diff
@@ -92,16 +100,24 @@ jobs:
         uses: xom9ikk/dotenv@v1.0.2
         with:
           path: .github/workflows/
+      # NOTE: The assignment rules has to be changed when the new column for automated PR is added
       - name: Assign to a person to work on it
-        # Only if nobody is assigned or moved into In progress FROM Triage
+        # Assign when there is nobody assigned or the card is new
+        if: ${{ needs.get-issue.outputs.assignees == '[]' || github.event.action == 'created' }}
+        uses: pozil/auto-assign-issue@v1.9.0
+        with:
+          numOfAssignee: 1
+          teams: ${{ github.event.project_card.column_id == env.BITNAMI_COLUMN_ID && env.SUPPORT_TEAM_NAME || env.TRIAGE_TEAM_NAME }}
+          repo-token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
+      - name: Reassign when moved into 'In progress' from 'Triage'
+        # Reassigned when moved into In progress FROM Triage
         if: |
-          needs.get-issue.outputs.assignees == '[]' ||
-          github.event.project_card.column_id == env.IN_PROGRESS_COLUMN_ID && github.event.changes != null && github.event.changes.column_id &&
+          github.event.action == 'moved' && needs.get-issue.outputs.assignees != '[]' &&
```

Comment:
If the issue was in triage and you move it, the assignees are not empty, not? Or am I missing anything? :/

Reply:
It shouldn't be empty. If assignees if empty the step `Assign to a person to work on it` will be executed and this one will be skipped